MapboxDirectionsTests/MapboxDirectionsTests.swift

```diff
@@ -26,29 +26,29 @@ class MapboxDirectionsTests: XCTestCase {
         var routes: [MBRoute] = []
         directions.calculateDirectionsWithCompletionHandler { (response, error) in
             XCTAssertNil(error, "Error: \(error)")
-            XCTAssertFalse(directions.calculating)
             
             XCTAssertNotNil(response)
             routes = response!.routes
             
+            
+            XCTAssertEqual(routes.count, 2)
+            XCTAssertEqual(routes.first!.geometry!.count, 28268)
+            
+            // confirming actual decoded values is important because the Directions API
+            // uses an atypical precision level for polyline encoding
+            XCTAssertEqual(round(routes.first!.geometry!.first!.latitude), 38)
+            XCTAssertEqual(round(routes.first!.geometry!.first!.longitude), -122)
+            XCTAssertEqual(routes.first!.legs.count, 1)
+            XCTAssertEqual(routes.first!.legs.first!.steps.count, 136)
+            XCTAssertEqual(routes.first!.legs.first!.steps[24].distance, 106_258)
+            XCTAssertEqual(routes.first!.legs.first!.steps[24].duration, 3_929)
+            XCTAssertEqual(routes.first!.legs.first!.steps[24].name, "I 80")
+            
             expectation.fulfill()
         }
-        waitForExpectationsWithTimeout(2) { (error) in
+        
+        waitForExpectationsWithTimeout(10) { error in
```

Comment:
Does the timeout need to be 10 seconds long? If so, it might be possible to reduce the time by moving the assertions back out of the completion handler.